README.md

```diff
@@ -39,7 +39,29 @@ Please file an issue if additional packages are required.
 * python3.9
 * python3.9-venv
 * musl-1.2.2
-* ARM GCC compiler; 10.2-2020.11
+* cmake
+* ninja-build
+* ARM GCC compiler for none-elf; 10.2-2020.11
```

Comment:
This should be version 12.2.1 20221205, my bad I forgot to update it in https://github.com/seL4/microkit/pull/94

Reply:
Cool I can fix that.

Reply:
Thanks